lib/qa/configuration.rb

```diff
@@ -37,5 +37,13 @@ def valid_authority_reload_token?(token)
     def default_language
       @default_language ||= :en
     end
+
+    # When true, prevents ldpath requests from making additional network calls.  All values will come from the context graph
+    # passed to the ldpath request.
+    attr_writer :limit_ldpath_to_context
+    def limit_ldpath_to_context?
+      return true if @limit_ldpath_to_context.nil?
```

Comment:
could be written as, which seems a bit simpler:
```
 @limit_ldpath_to_context ||= true
 @limit_ldpath_to_context
```

Reply:
That is how I had it originally, but if `@limit_ldpath_to_context == false`, it will see the `||=` check as not being set and change the value to true, thus causing `@limit_ldpath_to_context` to always be true.

Reply:
@elrayle you are so right!  Sorry!